.gas-snapshot

```diff
@@ -1,19 +1,28 @@
-COWShedFactoryTest:testDomainSeparators() (gas: 1111757)
-COWShedFactoryTest:testExecuteHooks() (gas: 596281)
-COWShedFactoryTest:testForwardResolve() (gas: 42016)
-COWShedFactoryTest:testReverseResolve() (gas: 38552)
-COWShedProxyTest:testAdmin() (gas: 1182051)
-COWShedProxyTest:testProxyInitialization() (gas: 1275503)
-COWShedProxyTest:testUpdateImplementation() (gas: 1187938)
-COWShedTest:testExecuteHooks() (gas: 182170)
-COWShedTest:testExecuteHooksDeadline() (gas: 47289)
-COWShedTest:testExecuteHooksDelegateCall() (gas: 105360)
-COWShedTest:testExecuteHooksForSmartAccount() (gas: 139099)
-COWShedTest:testRevokeNonce() (gas: 84678)
-COWShedTest:testTrustedExecuteHooks() (gas: 95100)
-COWShedTest:testUpdateImplementation() (gas: 28429)
-COWShedTest:testUpdateTrustedHook() (gas: 89955)
-LibAuthenticatedHooksTest:testCallHash() (gas: 13100)
-LibAuthenticatedHooksTest:testCallsHash() (gas: 13954)
-LibAuthenticatedHooksTest:testDecodeEOASignature(bytes32,bytes32,uint8) (runs: 257, μ: 11285, ~: 11285)
-LibAuthenticatedHooksTest:testExecuteHooksHash() (gas: 19485)
\ No newline at end of file
+COWShedFactoryTest:testDeploysExpectedProxyWithNoEns() (gas: 250639)
+COWShedFactoryTest:testExposesExpectedCreationCode() (gas: 10816)
+DeployTest:testMatchesOfficialAddresses() (gas: 2443967)
+DeployTest:testUsesCreate2() (gas: 2452754)
+ForkedCOWShedFactoryTest:testDeploymentFailsIfImplementationHasNoCode() (gas: 47126)
+ForkedCOWShedFactoryTest:testDomainSeparators() (gas: 1083256)
+ForkedCOWShedFactoryTest:testExecuteHooks() (gas: 577237)
+ForkedCOWShedFactoryTest:testExecuteHooksForRevertingEns() (gas: 413013)
+ForkedCOWShedFactoryTest:testForwardResolve() (gas: 41608)
+ForkedCOWShedFactoryTest:testInitializeProxyWithEns() (gas: 530793)
+ForkedCOWShedFactoryTest:testInitializeProxyWithoutEns() (gas: 273824)
+ForkedCOWShedFactoryTest:testReverseResolve() (gas: 37258)
+ForkedCOWShedProxyTest:testAdmin() (gas: 1272451)
+ForkedCOWShedProxyTest:testProxyClaimWithResolver() (gas: 1426085)
+ForkedCOWShedProxyTest:testProxyInitialization() (gas: 1366029)
```

Comment:
Thanks for providing the commit https://github.com/cowdao-grants/cow-shed/commit/e7f649cb18e3574297973b44b6cee28d62735ae8 

because this was outdated was confusing to review otherwise